.classpath

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
.classpath is an eclipse-specific file and we typically don't check those in. If you look at the .gitignore file in chpl (https://github.com/chpladmin/chpl-api/blob/development/.gitignore) you should see a bunch of Eclipse-specific files listed. You can probably copy that .gitignore here and use it without changing a thing.